### ISSUE TITLE
[$150]THRIVE - Graphic Design Trends That Will Become Popular in 2020

### ISSUE THREAD

**Issue by hasanozie** [topcoder-platform/thrive-submit#68]
Track Category: Design

[Hasanozie - Graphic Design Trends That Will Become Popular in 2020.docx](https://github.com/topcoder-platform/thrive-submit/files/3954922/Hasanozie.-.Graphic.Design.Trends.That.Will.Become.Popular.in.2020.docx)

**Comment by kkartunov** [comment]
Passed PCheck and min length check.

**Comment by Dara-K** [comment]
This is a good article, but it's more fit for the blog then THRIVE as it will have a short "life span" of about 5-6 months.

Can you improve the article for the blog? the blog articles are paid with $50.

If you can, here are a few things I see that need to be updated:
 - for each trend, let's use 1-2 images that are more relevant to app design or web design, rather than general since we want to give examples here (visual ones count too)
- can you add another 4-5 more design trends? there are so many online predicted, it's tricky to say which will be the dominant ones.

Thanks!

**Comment by Dara-K** [comment]
@kkartunov  Note: this article is fit for the blog, not THRIVE. It's up to the designer if he wishes to update or not.

**Comment by hasanozie** [comment]
Hi @Dara-K  ,
Thank you for reviewing this article
I will try to add content to this article so that it can be displayed on blog

**Comment by hasanozie** [comment]
Hi @Dara-K 

> for each trend, let's use 1-2 images that are more relevant to app design or web design, rather than general since we want to give examples here (visual ones count too)

may I use the designs of other designers on dribbble as supporting images of each of the trend lists?
because I have limited skills to make all these design trends myself.

**Comment by Dara-K** [comment]
Thank you!

Yes, of course. Just add the link to their dribbble page.

**Comment by hasanozie** [comment]
Hi @Dara-K 
I have changed all images to web/app design from dribbble and have added 4 design trends to this article.

[Hasanozie - Graphic Design Trends That Will Become Popular in 2020.docx](https://github.com/topcoder-platform/thrive-submit/files/3978057/Hasanozie.-.Graphic.Design.Trends.That.Will.Become.Popular.in.2020.docx)

**Comment by Dara-K** [comment]
This looks much better!

_The futuristic design will make everyone's imagination full of curiosity about the future so this design is perfect for representing the latest products or products that will be launched in the future. So you don't need to be afraid to apply futuristic design to the design project that you are working on._ 
Can you rephrase this to something better? "future "and "futuristic" repeat too many times.

That is the last update - it looks great otherwise. Thank you!

**Comment by hasanozie** [comment]
> The futuristic design will make everyone's imagination full of curiosity about the future so this design is perfect for representing the latest products or products that will be launched in the future. So you don't need to be afraid to apply futuristic design to the design project that you are working on.


Hi @Dara-K 
I have reduced the word "future" in the text above
[Hasanozie - Graphic Design Trends That Will Become Popular in 2020.docx](https://github.com/topcoder-platform/thrive-submit/files/3986192/Hasanozie.-.Graphic.Design.Trends.That.Will.Become.Popular.in.2020.docx)

**Comment by hasanozie** [comment]
Is the article on the Thrive Topcoder allowed to take design images from dribbble or behance? because it will make the article easier to understand than using stock photos from unsplash or pexels.

**Comment by Dara-K** [comment]
Yes, for design track we can allow that because it helps underline a point better.  So as long as you give credit to the author/link that's fine.

Looks better now, thanks!


I have removed the second sentence from the Futuristic trend as it was repetitive. Hope that's okay with you.
[Hasanozie.-.Graphic.Design.Trends.That.Will.Become.Popular.in.2020_V2.docx](https://github.com/topcoder-platform/thrive-submit/files/3988697/Hasanozie.-.Graphic.Design.Trends.That.Will.Become.Popular.in.2020_V2.docx)

**Comment by hasanozie** [comment]
> I have removed the second sentence from the Futuristic trend as it was repetitive. Hope that's okay with you.

No Problem, thanks for helping me :)

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30111606 has been created for this ticket.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30111606 has been updated - it has been assigned to hasanozie.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
@hasanozie can't access the banner image for this one. Please, make it available to me

**Comment by Dara-K** [comment]
@kkartunov  Please note that this article will go to the blog - not THRIVE.

**Comment by hasanozie** [comment]
Hi @kkartunov 
I have updated the banner image link for this article.

[Hasanozie.-.Graphic.Design.Trends.That.Will.Become.Popular.in.2020_V2.docx](https://github.com/topcoder-platform/thrive-submit/files/4003710/Hasanozie.-.Graphic.Design.Trends.That.Will.Become.Popular.in.2020_V2.docx)

**Comment by Dara-K** [comment]
Thanks @hasanozie ! I will share this with our editor to grammar check it and then we will publish it.

**Comment by hasanozie** [comment]
You're welcome @Dara-K 
I hope this article can be useful for all designers :)

**Comment by kkartunov** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30111606<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```
